README.md

```diff
@@ -716,11 +702,43 @@ _NOTE: Skip these steps if you are deploying without promoting changes._
    and resolve conflicts in necessary. Conflict resolution should only be
    necessary if cherry-picks occured on the target branch.
 
-4. The merge may have affected `README.md`, the file you are looking at right
+3. The merge may have affected `README.md`, the file you are looking at right
    now. Reopen the file now to ensure you are following the updated version.
 
+4. Now you need to create the release notes. (Skip this step if noA link to the release
+   notes document can be found either in the #dcp-ops channel on HCA Slack or in
+   the Google Drive folder mentioned in the [DCP release SOP].
+
+   To produce the list of changes for the DCP release notes, run
+
+   ```
+   git log --pretty=oneline --topo-order --abbrev-commit
+   ```
+
+   All non-merge commits from the top down to the commit labeled with the most
+   recent `deployed/…` tag for this deployment stage represent changes to be
+   deployed and should be mentioned in the release notes. Copy them over, but
+   remember to remove tag / branch names.
+
+   You will also need to add the release tag and commit hash which are generated
+   later in this guide.
+
+5. At this point you should determine whether or not you will need to reindex.
+   Look at the history of `CHANGELOG.yml` since the last release to determine
+   when reindexing is necessary. When in doubt assume yes. In that case run the
+   announce your conclusion in the 
+   [#data-wrangling](https://humancellatlas.slack.com/messages/C9ENBPVNW)
+   (this should be done before 5pm Pacific, Monday for staging and Tuesday for
+   production).
+
 ### 6.1.3 Finishing up deployment / promotion
 
+If promoting to staging or production this part of the process should be
+coordinated on the [#dcp-ops](https://humancellatlas.slack.com/messages/G9XD6L0AD)
+Slack channel. If also reindexing, send a second warning to the #data-wrangling
+channel once the go-ahead is received. This warning should happen ~20 minutes
+before actual reindexing.
```

Comment:
A typical builds takes 20min. That's why I chose that time. You would push the branch and then notify the channel.

Reply:
I'll make that more explicit.